.github/workflows/create-dev-tag.yml

```
@@ -0,0 +1,71 @@
+name: Create Development Tag
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: '5 20 * * *'
```

the teamcity trigger is set to 20:05 PST every day. but this cron is using UTC. 

I think we want to use this:

```suggestion
    - cron: '5 3 * * *'
```